usr/lib/common/lexer.l

```diff
@@ -69,6 +69,7 @@ extern char *configparse_strdup(const char *s);
 
 version                 return OCKVERSION;
 slot                    return SLOT;
+disable-event-support   return DISABLE_EVENT_SUPPORT;
```

Comment:
Technically that is a backward incompatible change, but lets hope nobody used the bare-word-string `disable-event-support` somewhere in the configuration before (e.g., as a description).  We should really at some point fix this parser to become more context sensitive...

Reply:
I cold certainly choose an even longer word for that to make it even more unlikely that it is already use somewhere :-) Suggestions are welcome.....

Reply:
No, the real solution is a fix in the lexer to move to a proper lexing mode and a change to the parser to use this mode at appropriate places.  So also no change here.  The problem is very theoretical (which I forgot to write in my initial comment)...